libraries/src/Application/SiteApplication.php

```diff
@@ -56,6 +56,14 @@ final class SiteApplication extends CMSApplication
      * @since  4.0.0
      */
     protected $detect_browser = false;
+ 
+    /**
```

Comment:
```suggestion

    /**
```
Fix PHPCS error "Whitespace found at end of line" for line 59.